solutions/src/test/java/ru/sberbank/school/task10/ScalableThreadPoolTest.java

```diff
@@ -0,0 +1,68 @@
+package ru.sberbank.school.task10;
+
+import lombok.RequiredArgsConstructor;
+import org.junit.jupiter.api.Test;
+
+class ScalableThreadPoolTest {
+    private final ScalableThreadPool threadPool;
+
+    {
+        threadPool = new ScalableThreadPool(2, 5);
+        threadPool.startMonitoringDaemon();
+    }
+
+    @Test
+    void startFewTasksTest() throws Exception {
+        int tasks = 20;
+        int tasksIntervalMs = 500;
+        int workDurationMs = 100;
+        startTest(tasks, tasksIntervalMs, workDurationMs);
+    }
+    @Test
```

Comment:
&#39;METHOD_DEF&#39; should be separated from previous statement.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1578027451>*